utils/tests/artifactoryconsts.go

```diff
@@ -1430,8 +1430,8 @@ func GetReplicationConfig() []clientutils.ReplicationParams {
 	return []clientutils.ReplicationParams{
```

Comment:
What do you say we take this opportunity and rename the tests package to tests_test, so that it is not packages as part of the binary?

Reply:
See my comment here: https://github.com/jfrog/jfrog-cli/pull/770#discussion_r450667703